src/cdi/CDI.hh

```diff
@@ -201,7 +201,7 @@ static double Planck2Rosseland(double const freq, double const exp_freq) {
   double factor;
 
   if (freq > 1.0e-5)
-    factor = NORM_FACTOR * (freq_3 * freq) / std::expm1(freq);
+    factor = NORM_FACTOR * (exp_freq * freq_3) * (freq / -std::expm1(-freq));
   else
```

Comment:
Hey - As soon as I sent this to you I realized that my DbC check is bad.  Try this instead:

```
Check( freq>1.0 ? freq < std::numeric_limits<double>::max()/freq_3 : true );
```

If `1.0e-5 < freq < 1.0`, there is no risk of overflow (or underflow).